test/Microsoft.IdentityModel.Tokens.Saml.Tests/Saml2SecurityTokenHandlerTests.cs

```diff
@@ -14,7 +14,7 @@
 
 namespace Microsoft.IdentityModel.Tokens.Saml2.Tests
 {
-    public class Saml2SecurityTokenHandlerTests
+    public partial class Saml2SecurityTokenHandlerTests
```

Comment:
Why do we need a partial test class?

Reply:
Would it be better to name the other test class something more specific to what it tests?

Reply:
This is to keep every aspect of our new validations separate from what we currently have under our old call path. Also, helps us keep track of each validation we complete as, we have to be very thorough in testing our new functionality so, helps reduce noise from other non-relevant tests. Once we ship, we can consider consolidating them into one file with a single name, should be pretty simple to do.